### DISCUSSION TITLE
joint stereo ?

### DISCUSSION THREAD

**Discussion by ghost** [Tichau/FileConverter#373] [Ideas]
Can we add the joint stereo in addition to the stereo and mono to the mp3 format?
Thanks in advance

**Comment by Tichau** [comment]
You can look at the ffmpeg documentation and create an advanced preset to do this. 

I'm starting to create a library of custom preset here: https://github.com/Tichau/FileConverter/tree/integration/PresetSamples
So we can collect, as a community, the very specific preset that we need.

**Comment by ghost** [comment]
THANKS.
If I understood correctly, the lines of code to be added are :
```
<Settings Key="EnableFFMPEGCustomCommand" Value="True" />
<Settings Key="FFMPEGCustomCommand" Value="-vn -ac 2 -joint-steo 1" />
```

**Reply by Tichau** [reply]
Yeah that seems to be good !

You can export the preset you have done by right clicking on it then selecting 'Export', 
then put the file you've exported in the 'PresetSamples' and create a pull request.
